consul-secure-acl-vault-hashiconf/step1.md

```diff
@@ -0,0 +1,40 @@
+
+The Consul secrets engine generates Consul API tokens dynamically based on 
+Consul ACL policies, and must be enabled before it can perform its functions.
```

Comment:
```suggestion
The Consul secrets engine generates Consul API tokens in Vault dynamically based on 
Consul ACL policies, and must be enabled before it can perform its functions.
```
Clarifies what process is generating the Vault tokens.. I could be wrong here.. maybe it should be "Vault's Consul secrets engine.." instead